avida-ts/docker/docker-compose.local.yaml

```diff
@@ -0,0 +1,82 @@
+version: "3"
+
+services:
+  cheqd-node:
+    image: ghcr.io/nymlab/cheqd-node:v2.0.1-arm64
+    ports:
+      - target: 9090
+        published: 9090 # grpc
+      - target: 26656
+        published: 26656 # Tendermint peer-to-peer (P2P) port
+      - target: 26657
+        published: 26657 # Tendermint RPC port
+    volumes:
+      - ./scripts/:/tmp
+    entrypoint: /tmp/cheqd-genesis.sh
+    env_file:
+      - ../local.env
+
+  cheqd-resource-uploader:
+    image: ghcr.io/nymlab/cheqd-node:v2.0.1-arm64
+    ports:
+      - target: 26656
```

Comment:
I think we don't need to expose ports here cause we do not run the node inside the container.

Reply:
updated in https://github.com/nymlab/AVIDA/pull/6/commits/7ce3c1886296d88d19edcfd650036b81c8dcdf75